src/components/HomeGuestsloves.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import HotelItem from './HotelItem';
+import hotelData from '../constants/HotelsData';
+import Slider from 'react-slick';
+
+
+export default class HomeGuestsloves extends Component {   
+  render() {
+    const settings = {
+      infinite: true,
+      speed: 500,
+      slidesToShow: 4,
+      slidesToScroll: 4
+    };
+    
```

Comment:
I would also try adding styles to the arrows, as they don't look quite what we expect. To do this, you can refer to the documentation(https://react-slick.neostack.com/docs/example/custom-arrows) and create ```nextArrow:``` and ```prevArrow:``` , or just see what classes they have and add styles directly via the .css file(.slick-prev and .slick-next)

![image](https://user-images.githubusercontent.com/20189313/145734983-b1d4fec7-52c8-4ca0-a762-3bf015a3964f.png)